client/ayon_core/hosts/photoshop/plugins/create/create_image.py

```diff
@@ -53,7 +53,7 @@ def create(self, product_name_from_ui, data, pre_create_data):
             stub.select_layers(stub.get_layers())
             try:
                 group = stub.group_selected_layers(product_name_from_ui)
-            except:
+            except:  # noqa E722
```

Comment:
Maybe we can chain it like this. I think it is only `ValueError` here we are interested it.
```suggestion
            except ValueError as e:
                raise CreatorError(e) from e
```

Reply:
Moved ValueError exception higher.